requirements_test_all.txt

```diff
@@ -104,6 +104,9 @@ prometheus_client==0.0.19
 # homeassistant.components.zwave
 pydispatcher==2.0.5
 
+# homeassistant.components.climate.ephember
+pyephember==0.1.1
```

Comment:
Was this added by a script or did you add it manually?

Reply:
I used script/gen_requirements_all.py to add it

Reply:
Caused by a bug in the script. I sent PR #9764 to fix it

Reply:
Should I remove it here?

Reply:
No, the file shouldn't be edited manually.

Reply:
I submitted the fix. Could you sync to HEAD and re-run the script?